trustmanager/filestore.go

```diff
@@ -15,40 +15,33 @@ type SimpleFileStore struct {
 	perms   os.FileMode
 }
 
-// NewSimpleFileStore creates a directory with 755 permissions
-func NewSimpleFileStore(baseDir string, fileExt string) (*SimpleFileStore, error) {
+// NewFileStore creates a fully configurable file store
+func NewFileStore(baseDir, fileExt string, perms os.FileMode) (*SimpleFileStore, error) {
```

Comment:
nit: should this method be unexported, and only be accessible from `NewSimpleFileStore` and `NewPrivateSimpleFileStore`?